draft-ietf-teep-architecture.md

```diff
@@ -1134,21 +1123,22 @@ for protecting the resource usage allocated for TA management.
 
 ## Compromised CA
 
-A root CA for TAM certificates might get compromised. A Trust Anchor other 
-than a root CA certificate may also be compromised. Some TEE Trust
+TAMs are responsible for validating the TEE certificate chain, including
+the TEE certificate and any intermediate CA certificates up to the root
+certificate. Such validation includes checking for certificate revocation.
+A root CA for TAM certificates might get compromised or its certificate might
+expire. A Trust Anchor other 
+than a root CA certificate may also expire or be compromised. 
+If the Trust Anchor used by a TEE device certificate expires or is revoked, the
+device might be rejected by a TAM, subject to the TAM's policy.
+To address these, some TEE Trust
 Anchor update mechanism is expected from device OEMs. 
```

Comment:
The original paragraph is to say that TEEP Agent stores Trust Anchors to validate TAM certificates. If some Trust Anchor is compromised, the device needs to update them. Therefore some "TEE Trust Anchor update mechanism is expected" to "update the list in the devices. I think these last two lines should be moved to the next paragraph below that talks about TEEs validating TAM certificates.

Reply:
Please check updated text